### ISSUE TITLE
Add Fedora 26 to to all branches when it is released

### ISSUE THREAD

**Issue by rallytime** [saltstack/salt-jenkins#318]
@Ch3LL commented on [Wed Apr 26 2017](https://github.com/saltstack/zh/issues/1269)

---

@rallytime commented on [Wed Apr 26 2017](https://github.com/saltstack/zh/issues/1269#issuecomment-297529900)

We actually can't add this yet. Since Fedora 26 won't be released for a while (June?) there isn't an image available yet in Linode to use for tests.

@Ch3LL Should we close this one, or keep it open and move it to backlog or blocked for a later date?

---

@Ch3LL commented on [Thu Apr 27 2017](https://github.com/saltstack/zh/issues/1269#issuecomment-297714860)

Totally agree. Moving this to a different epic since its not dependent on the nitrogen release.

We can do this once fedora 26 is out of alpha later on

**Comment by rallytime** [comment]
This is a reminder issue to add Fedora 26 to all current release branches when Fed26 released, likely in June.

**Comment by rallytime** [comment]
This should probably be on @dubb-b's radar. Fed26 still hasn't bee released yet though. Looks like current schedules is around mid-July. I'm not sure when Linode will provide the image or when we'll have package support available.

**Comment by dmurphy18** [comment]
Fedora 26 Final Release Readiness Meeting on Thursday, July 6th @ 19:00 UTC

so maybe by end of week after Independence Day

**Comment by rallytime** [comment]
Looks like Fed 26 is out now. There are images in Linode now. Also, I just tested the current stable bootstrap script and it works just fine out of the box to install git. No additions are necessary to get these tests running. 

Note, however, that you'll need to run `-P` as a script argument when installing via git for jenkins.
I am looking into removing this requirement for the future, but the current bootstrap script will need it. This is because newer versions of tornado were not available on older Fedoras, but Fedora 26 has a more recent version available.

@dubb-b FYI ^

**Comment by rallytime** [comment]
The next stable version of the bootstrap script will no longer require `-P`. I've fixed this requirement here: https://github.com/saltstack/salt-bootstrap/pull/1114.

**Comment by dubb-b** [comment]
@rallytime The plans are all added, when you merge forward I can validate that they are running. If there are any other issues I can fix.

**Comment by rallytime** [comment]
@dubb-b I just did a merge forward from 2017.7 to the master branch in salt-jenkins.

Can you add `Fedora 26` to the `develop` branch tests when you get a chance? Once that is done, we can close this.

**Comment by dubb-b** [comment]
@rallytime Minus the issues with the tests this seems to be complete on my end. Let me know if you need any params changed.

**Comment by rallytime** [comment]
Tests are running so it looks good to me. We'll have to figure out why that many tests are falling down, but that will be a new issue. :)
